enable.ps1

```diff
@@ -64,183 +57,175 @@ function Resolve-InceptionError {
-            }
+        if (-not [string]::IsNullOrEmpty($ErrorObject.ErrorDetails.Message)) {
+            $httpErrorObj.ErrorDetails = $ErrorObject.ErrorDetails.Message
         }
-        if ($webresponse) {
-            try {
-                $convertedErrorObject = ($httpErrorObj.FriendlyMessage | ConvertFrom-Json)
-                if (-not [string]::IsNullOrEmpty($convertedErrorObject.languageString)) {
-                    $httpErrorObj.FriendlyMessage = $convertedErrorObject.LanguageString
-                }
-                elseif (-not [string]::IsNullOrEmpty($convertedErrorObject.description)) {
-                    $httpErrorObj.FriendlyMessage = $convertedErrorObject.Description
+        elseif ($ErrorObject.Exception.GetType().FullName -eq 'System.Net.WebException') {
+            if ($null -ne $ErrorObject.Exception.Response) {
+                $streamReaderResponse = [System.IO.StreamReader]::new($ErrorObject.Exception.Response.GetResponseStream()).ReadToEnd()
+                if (-not [string]::IsNullOrEmpty($streamReaderResponse)) {
+                    $httpErrorObj.ErrorDetails = $streamReaderResponse
                 }
             }
-            catch {
-                Write-Warning "Unexpected webservice response, Error during Json conversion: $($_.Exception.Message)"
+        }
+        try {
+            $errorDetailsObject = ($httpErrorObj.ErrorDetails | ConvertFrom-Json)
+            # Make sure to inspect the error result object and add only the error message as a FriendlyMessage.
+            if (-not [string]::IsNullOrEmpty($errorDetailsObject.languageString)) {
+                $httpErrorObj.FriendlyMessage = $errorDetailsObject.LanguageString
             }
+            elseif (-not [string]::IsNullOrEmpty($errorDetailsObject.description)) {
+                $httpErrorObj.FriendlyMessage = $errorDetailsObject.Description
+            }            
+            $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails # Temporarily assignment
+        }
+        catch {
+            $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails
         }
         Write-Output $httpErrorObj
     }
 }
 #endregion
 
-$accountUser = [PSCustomObject]@{
-    id       = $aRef.AccountReference
-    type     = 14
-    name     = $p.Contact.Business.Email # UPN
-    password = (Get-RandomCharacters -length 10 -characters 'abcdefghijklmnopqrstuvwxyzABCDEFGHKLMNOPRSTUVWXYZ1234567890!@#%&+{}') # Only used with new creations. Proprety is mandatory but is not used when using SSO.
-}
-
-# Begin
 try {
-    Write-Verbose "Verifying if a Inception account for [$($p.DisplayName)] exists"
+    # Verify if [aRef] has a value
+    if ([string]::IsNullOrEmpty($($actionContext.References.Account))) {
+        throw 'The account reference could not be found'
+    }
+
+    Write-Verbose "Verifying if an Inception account for [$($personContext.Person.DisplayName)] (still) exists"
+    #$correlatedAccount = 'userInfo'
```

Comment:
$correlatedAccount = 'userInfo' can be removed in the scripts (left over from template)

Reply:
Removed